cmd/lncli/cmd_open_channel.go

```diff
@@ -103,8 +103,9 @@ var openChannelCommand = cli.Command{
 		cli.BoolFlag{
 			Name: "fundmax",
 			Usage: "(optional) if set, the wallet will attempt to " +
-				"commit the maximum possible local amount to the channel. " +
-				"Should not be set together with local_amt",
+				"commit the maximum possible local amount to " +
```

Comment:
This whole commit should be squashed with the respective commits where the initial changes were introduced.

Reply:
Done here https://github.com/lightningnetwork/lnd/pull/6903/commits/b8e351e04a948152dbfd8cd965848c47619d2d0c and here
https://github.com/lightningnetwork/lnd/pull/6903/commits/7915d0dd52683ebecc291843ccc802079d333a48